pkgs/development/tools/misc/kibana/default.nix

```diff
@@ -1,14 +1,23 @@
 { stdenv, makeWrapper, fetchurl, nodejs, coreutils, which }:
 
 with stdenv.lib;
-
-stdenv.mkDerivation rec {
+let
+  inherit (builtins) elemAt;
+  info = splitString "-" stdenv.system;
+  arch = elemAt info 0;
+  plat = elemAt info 1;
+  shas = {
```

Comment:
Lookups on this attrset will fail to evaluate on i686-linux. Either add an `or throw ...` to the attrset lookup result or an assert at the top.